plugins/services/src/js/components/ServiceStatusIcon.js

```diff
@@ -118,32 +117,48 @@ class ServiceStatusIcon extends Component {
 
   render() {
     const { service } = this.props;
-    const displayName = service.getServiceStatus().displayName;
-    let iconState = displayName;
-
-    if (
-      displayName === ServiceStatusLabels.DEPLOYING ||
-      displayName === ServiceStatusLabels.RECOVERING ||
-      displayName === ServiceStatusLabels.DELETING
-    ) {
-      iconState = "TRANSITION";
+    const serviceStatus = service.getServiceStatus();
+    let iconState;
+
+    switch (serviceStatus) {
```

Comment:
I'm wondering if I should `if` statements instead that way I can combine the 3 `LOADING` checks in one if and save a couple lines.

Reply:
@weblancaster I think that makes sense!

Reply:
But I'm also good with it as is :)

Reply:
I'm also good as it is now